assets/js/vue/utils/template-switch.js

```diff
@@ -0,0 +1,26 @@
+const grp = {
+    status: 'Dataset Status',
+    fundingCycle: 'Grant Awards',
+    researchGroup:  'Projects',
+    fundingOrg: 'Funding Organizations'
+};
+
+const griidc = {
+    status: 'Dataset Status',
+    fundingCycle: 'Funding Cycles',
+    researchGroup:  'Research Groups',
+    fundingOrg: 'Funding Organizations'
+}
+
+export default {
+    getLabel: (name) => {
+        if (PELAGOS_TEMPLATE_PROPS === undefined || PELAGOS_TEMPLATE_PROPS.BaseTemplateName === undefined || PELAGOS_TEMPLATE_PROPS.BaseTemplateName === 'GRIIDC') {
```

Comment:
undefined doesn't mean GRIIDC. It's means not GRP. What if have another BaseTemplateName of something else, in this case it would assume it's GRP.

Reply:
So if it's not GRP, and we don't know what it is, how should we handle it? I thought GRIIDC was always the fallback.

Reply:
Yes, GRIIDC is default, maybe it should be called that. Maybe up front say Template is GRIIDC until I know otherwise.